docs/references/ic-interface-spec.md

```diff
@@ -779,7 +785,7 @@ The HTTP response to this request can have the following responses:
 
 -   5xx HTTP status when the server has encountered an error or is otherwise incapable of performing the request. The request might succeed if retried at a later time.
 
-This request type can *also* be used to call a query method (but not a composite query method). A user may choose to go this way, instead of via the faster and cheaper [Request: Query call](#http-query) below, if they want to get a *certified* response. Note that the canister state will not be changed by sending a call request type for a query method (except for cycle balance change due to message execution).
+This request type can *also* be used to call a query method (but not a composite query method). A user may choose to go this way, instead of via the faster and cheaper [Request: Query call](#http-query) below, if they want to get a *certified* response. Note that the canister state will not be changed by sending a call request type for a query method (except for transient state such as cycle balance, canister logs, and canister version).
```

Comment:
Is there a difference between sync and async calls? If not, can we use the exact same phrasing twice. 

> (except for cycle balance change due to message execution).
or

> (except for transient state such as cycle balance, canister logs, and canister version).

Reply:
> Is there a difference between sync and async calls?

The question was about sync (/api/v3/.../call or /api/v4/.../call) vs async (/api/v2/.../call) calls, not about update (/api/.../call) vs query (/api/.../query).